selvpc/networking.go

```diff
@@ -40,10 +40,28 @@ func subnetsMapsFromStructs(subnetsStructs []subnets.Subnet) []map[string]interf
 	associatedSubnets := make([]map[string]interface{}, len(subnetsStructs))
 
 	for i, subnet := range subnetsStructs {
-		associatedSubnets[i] = map[string]interface{}{
-			"network_id": subnet.NetworkID,
-			"subnet_id":  subnet.SubnetID,
-			"region":     subnet.Region,
+		associatedSubnets[i] = make(map[string]interface{})
+
+		if subnet.NetworkID != "" {
```

Comment:
We don' need those checks.
I think that we need to populate `subnets` attribute Set with all computed fields even if they have zero values.

Also, this will create a panic in `resource_selvpc_resell_crossregion_subnet_v2.go` L#186 (I left another comment there) if we don't populate the `project_id` key.